src/web/app/pages-session/session-result-page/session-result-page.component.html

```diff
@@ -1,19 +1,35 @@
 <h1>
   Feedback Session Results
 </h1>
-<div class="card bg-light top-padded" *ngIf="session">
-  <div class="card-body row" style="padding: 40px;">
-    <label class="col-sm-2 control-label">Course ID:</label>
-    <div class="col-sm-10">{{ session.courseId }}</div>
+<div class="card card-plain" ngtif="session">
+  <div class="card-body">
+    <br/>
+    <div class="row">
+      <div class="col-2 text-right"><b>Course ID:</b></div>
+      <div class="col-5">{{ session.courseId }}</div>
+    </div>
+    <br/>
 
-    <label class="col-sm-2 control-label">Session:</label>
-    <div class="col-sm-10">{{ session.feedbackSessionName }}</div>
+    <br/>
```

Comment:
The extra `</br>` here is causing the lines to have unnecessary spacing than what is usually needed:
![image](https://user-images.githubusercontent.com/25320782/56456694-601ee500-63a2-11e9-915d-2b77860d0009.png)